### ISSUE TITLE
Index organismQuantity, organismQuantityType, sampleSizeUnit, sampleSizeValue

### ISSUE THREAD

**Issue by thomasstjerne** [gbif/pipelines#244]
It would be helpful for users to be able to filter out e.g. `organismQuantity < 2` as for metagenomic studies, this might be errors or contaminations.

It would also be extremely helpful to have a calculated field on occurrences where the value of `organismQuantityType ` is equal to the value of `sampleSizeUnit ` and there is an integer in both `organismQuantity` and `sampleSizeValue `.

Then it is possible to calculate the `organismQuantity` relative to the `sampleSizeValue ` i.e.
`organismQuantity / sampleSizeValue`.
See e.g. [this MGnify occurrence](https://api.gbif.org/v1/occurrence/2013991094/fragment) where the value of both `organismQuantityType` and `sampleSizeUnit` is `DNA sequence reads`

To summarize, please index:

1. `organismQuantity`
2. `organismQuantityType`
3. `sampleSizeUnit`
4. `sampleSizeValue`
5. If possible also `organismQuantity / sampleSizeValue` the name for this could be `relativeOrganismQuantity`

**Comment by muttcg** [comment]
Implemented in pipelines version 2.3.2, deployed in DEV
